independent-projects/bootstrap/maven-resolver/src/main/java/io/quarkus/bootstrap/resolver/maven/options/BootstrapMavenOptions.java

```diff
@@ -101,16 +110,92 @@
     }
 
     public static BootstrapMavenOptions newInstance() {
-        return newInstance(getMavenCmdLine());
+        return newInstance(getMavenCmdLine(false).cmdLine);
     }
 
     public static BootstrapMavenOptions newInstance(String cmdLine) {
         return new BootstrapMavenOptions(parse(cmdLine));
     }
 
-    public static String getMavenCmdLine() {
```

Comment:
I don't think this method should be removed. It could simply delegate to getMavenCmdLine(true), which I would make return a string.